<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scopeWidget</class>
 <widget class="QWidget" name="scopeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB signal</string>
  </property>
  <widget class="QwtPlot" name="dabScope">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Spectrum display, shows the frequency spectrum from the channel currently being received.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="scopeAmplification">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>18</width>
     <height>201</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QwtPlot" name="iqDisplay">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Constellation digram, obtained from the first data block in a dab frame. Ideally there are 4 points, one in each corner. The quality of the signal is inverse to the clouds that are visible,&lt;/p&gt;&lt;p&gt;i.e. lots of clouds =&amp;gt; poor signal.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="scopeSlider">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>160</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
